lnst/Recipes/ENRT/MeasurementGenerators/BaseFlowMeasurementGenerator.py

```diff
@@ -19,6 +19,7 @@
 MEASUREMENT_LOOKUP = {
     'iperf': IperfFlowMeasurement,
     'neper': NeperFlowMeasurement,
+    'xdp-bench':XDPBenchMeasurement,
```

Comment:
so since this class REQUIRES the use of an `xdp_command` it probably shouldn't be listed here, but only available in the derived class - if you include it here, then all test recipes that inherit from here gain the ability to specify the use of the xdp-bench tool as a measurement tool and since they don't define the parameter the instance init will crash.